compiler/codegen/ELFObjectFileGenerator.cpp

```diff
@@ -21,6 +21,10 @@
 
 #if defined(LINUX)
 
+#if defined(OMRZTPF)
+#include <stdio.h>
+#endif /* defined(OMRZTPF) */
```

Comment:
Can you just include `stdio.h` unconditionally?  This source file does use features found in stdio.h, so it's not wrong to include it regardless of the build target.  Or am I missing something?

Reply:
@0xdaryl So the only reason I added our platform guard macro was so that someone in the future would think twice about removing it but I think that is highly unlikely someone would remove stdio.h.

I removed the guard macros per your request.  Thanks.!